### ISSUE TITLE
Improve airport expenses section.

### ISSUE THREAD

**Issue by arv187** [nabeelio/phpvms#776]
**Is your feature request related to a problem? Please describe.**
1. Expenses don't admit decimals
2. In subfleet there are an option to 

> Cost Delay Per Minute.

**Describe the solution you'd like**
Permit decimals for quantities minors to 1
I think that this options is more appropriate in Airport editor to fix the tax for stay off the block.

**Describe alternatives you've considered**
Add more expenses options (actually there are flight, daily, monthly).
Time (minutes)
Per passenger
Per airplane weight (Mtow)

**Additional context**
![expenses](https://user-images.githubusercontent.com/16170732/88543643-8fe28100-d018-11ea-98ef-94aa904fea9a.png)

**Comment by nabeelio** [comment]
So you'd like per-minute, per pax and per weight unit expenses?

**Comment by arv187** [comment]
Yes, Because airport taxes are mainly related to the mtow of the aircraft, per passenger transported and per time spent using the facilities (aircraft parking)

**Comment by FatihKoz** [comment]
Just an addition to this request (which is really good to have, thx arv187) ... Expense per time (mins) should be applicable to flights too , with this enabled we can add **ATC Charges** to flights according to the **actual flight time**. It is apprx around 6-8 USD per min over Europe (depends on the country but we can use a rounded value for all at the moment)
